### ISSUE TITLE
Add YOLOv5x coco pretrained checkpoint

### ISSUE THREAD

**Issue by deanmark** [Deci-AI/super-gradients#231]
Is it possible to add a YOLOv5x coco pretrained checkpoint?

**Comment by oferbaratz** [comment]
Hi,
Currently, we are working on adding YOLO-X architecture into the repo (pre-trained checkpoints and recipes)
As a part of this effort we are training YOLO-X X as well

**Comment by shaydeci** [comment]
Im closing this issue down as we removed YoloV5 models from our repository.
As discussed above, YoloX L model, pretrained weights and training recipe are now available.

**Comment by shaydeci** [comment]
Im closing this issue down as we removed YoloV5 models from our repository.
As discussed above, YoloX L model, pretrained weights and training recipe are now available.
